cashweb-registry/src/store/topics.rs

```diff
@@ -56,22 +57,26 @@ pub enum DbTopicsError {
 }
 use self::DbTopicsError::*;
 
+use super::db::CF_BURNS;
```

Comment:
Always use `crate::` imports; plus move this to the import section at the top of the file

Reply:
This is what rust-analyzer generated, but will look.